src/system/MailerModule/Resources/views/Config/config.html.twig

```diff
@@ -0,0 +1,62 @@
+{% form_theme form with [
+    'ZikulaFormExtensionBundle:Form:bootstrap_3_zikula_admin_layout.html.twig',
+    'ZikulaFormExtensionBundle:Form:form_div_layout.html.twig'
+] %}
+{{ adminHeader() }}
+<div class="container-fluid">
+    <div class="row">
+        <h3>
+            <span class="fa fa-wrench"></span>
+            {{ __('Settings') }}
+        </h3>
+
+        {# http://swiftmailer.org/docs/sending.html #}
+        {% if zikulamailermodule_hasProcessControlFunctions() != true %}
+            <div class='alert alert-danger'>
+                <h3>{gt text='WARNING'}</h3>
+                <p>{{ __f('The PHP functions, %s, that support SMTP and SENDMAIL do not appear to be functioning on this PHP installation. Please compile them and restart your server.', {'%s': '<code>proc_*</code>'}) }}</p>
+            </div>
+        {% endif %}
+        <p class="alert alert-info">
+            {{ __f('Mailer relies on %swiftmailer', {'%swiftmailer': '<a href="http://symfony.com/doc/current/reference/configuration/swiftmailer.html"><i class="fa fa-external-link"></i> SwiftMailer configuration</a>'})|raw }}
+        </p>
+
+        {{ form_start(form) }}
+        {{ form_errors(form) }}
+
+        <fieldset>
+            <legend>{{ __('General settings') }}</legend>
+            {{ form_row(form.transport) }}
+            {{ form_row(form.charset) }}
+            {{ form_row(form.encoding) }}
+            {{ form_row(form.html) }}
+            {{ form_row(form.wordwrap) }}
+            {{ form_row(form.enableLogging) }}
+        </fieldset>
+        <fieldset data-switch="zikulamailermodule_config[transport]" data-switch-value="smtp">
```

Comment:
I see this is part of `zikula-bootstrap.js` which I don't think you have included. You also might investigate if this is possible to accomplish with bootstrap's `.collapse()` js.